corehq/apps/users/views/mobile/users.py

```diff
@@ -743,6 +748,9 @@ def post(self, request, *args, **kwargs):
             if 'location_id' in request.GET:
                 user.set_location(loc)
 
+            if phone_number:
+                initiate_sms_verification_workflow(user, phone_number)
```

Comment:
regarding the return value, I think it's okay to skip that for now, as this is a one-off custom thing that we plan on redoing in the future.